yt/yt/server/node/exec_node/job_environment.cpp

```diff
@@ -1102,6 +1102,11 @@ class TCriJobEnvironment
         }
         spec->Credentials.Gid = ::getgid();
 
+        if (!config->GpuIndexes.empty()) {
+            spec->Environment["NVIDIA_DRIVER_CAPABILITIES"] = "all";
```

Comment:
Let's add it to the node dynamic config to be able to tune it easily in case of any problems.